model/faas/spans.yaml

```diff
@@ -35,9 +36,10 @@ groups:
       - ref: faas.document.time
       - ref: faas.document.name
 
-  - id: span.faas.timer
+  - id: span.faas.timer.server
```

Comment:
FaaS conventions recommend using `SERVER` for all incoming invocations

https://github.com/open-telemetry/semantic-conventions/blob/4f77620fe731c10d40f7d50c543d4e5c73a46ebf/docs/faas/faas-spans.md?plain=1#L130